app/package.json

```diff
@@ -21,6 +21,7 @@
     "react": "18.1.0",
     "react-native": "0.70.6",
     "react-native-auth0": "^2.17.1",
+    "react-native-config": "^1.5.0",
```

Comment:
I took a look at the package on `npm`, it's fairly popular at `200k` DLs per week. I think that should be OK? What do you guys think? I know that DLs per week isn't really a good metric of how well supported a package will be in the future, but it sort of works.

Reply:
I think it's ok if it satisfies our need.